components/common/SocialLinks.tsx

```diff
@@ -42,16 +41,12 @@ function SocialLinks({
       {validLinks.map(link => {
         const Logo = LINK_LOGO[link];
         return (
-          <a
-            aria-label={`Go to ${siteConfig.name}'s ${link} profile`}
```

Comment:
Announcing "Got to" for each link is a bit redundant/annoying for screenreader users.

Reply:
you're right, even repeating "Scriptified's" is also redundant and quite obvious. 

just `"${link} profile"` looks appropriate to me. thoughts @prateek3255?